### PR TITLE
bug 1609680 updating docker image list

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1609680

@vikram-redhat  @ganhuang

### CODE REVIEW

File: install/disconnected_install.adoc [hunk 1]
```diff
@@ -195,24 +195,50 @@ endif::[]
 +
 [source, bash]
 ----
+$ docker pull registry.access.redhat.com/openshift3/apb-base:<tag>
+$ docker pull registry.access.redhat.com/openshift3/apb-tools:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: These two should be for the service catalog section.

File: install/disconnected_install.adoc [hunk 2]
```diff
@@ -195,24 +195,50 @@ endif::[]
...
+$ docker pull registry.access.redhat.com/openshift3/csi-attacher:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-driver-registrar:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-livenessprobe:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/image-inspector:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-maven-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: These two should be for section "Pull the Red Hat-certified Source-to-Image (S2I) builder images".

File: install/disconnected_install.adoc [hunk 3]
```diff
@@ -195,24 +195,50 @@ endif::[]
...
+$ docker pull registry.access.redhat.com/openshift3/local-storage-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/logging-eventrouter:<tag>
+$ docker pull registry.access.redhat.com/openshift3/manila-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/mariadb-apb:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: These two should be for the service catalog section.

File: install/disconnected_install.adoc [hunk 4]
```diff
@@ -195,24 +195,50 @@ endif::[]
...
+$ docker pull registry.access.redhat.com/openshift3/csi-attacher:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-driver-registrar:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-livenessprobe:<tag>
+$ docker pull registry.access.redhat.com/openshift3/csi-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/image-inspector:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-maven-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/local-storage-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/logging-eventrouter:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: This one should be for section "additional centralized log aggregation and metrics aggregation components"

File: install/disconnected_install.adoc [hunk 5]
```diff
@@ -195,24 +195,50 @@ endif::[]
...
+$ docker pull registry.access.redhat.com/openshift3/manila-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/mariadb-apb:<tag>
+$ docker pull registry.access.redhat.com/openshift3/metrics-schema-installer:<tag>
+$ docker pull registry.access.redhat.com/openshift3/mysql-apb:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: This one should be for the service catalog section.

File: install/disconnected_install.adoc [hunk 6]
```diff
@@ -195,24 +195,50 @@ endif::[]
...
+$ docker pull registry.access.redhat.com/openshift3/csi-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/efs-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/image-inspector:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-maven-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/jenkins-agent-nodejs-8-rhel7:<tag>
+$ docker pull registry.access.redhat.com/openshift3/local-storage-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/logging-eventrouter:<tag>
+$ docker pull registry.access.redhat.com/openshift3/manila-provisioner:<tag>
+$ docker pull registry.access.redhat.com/openshift3/mariadb-apb:<tag>
+$ docker pull registry.access.redhat.com/openshift3/metrics-schema-installer:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: Two `metrics-schema-installer` images present in this section. And this one should be for section "additional centralized log aggregation and metrics aggregation components".

File: install/disconnected_install.adoc [hunk 7]
```diff
@@ -195,24 +195,50 @@ endif::[]
...
 $ docker pull registry.access.redhat.com/openshift3/ose-f5-router:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-haproxy-router:<tag>
+$ docker pull registry.access.redhat.com/openshift3/ose-hyperkube:<tag>
+$ docker pull registry.access.redhat.com/openshift3/ose-hypershift:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-keepalived-ipfailover:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-pod:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-docker-builder:<tag>
+$ docker pull registry.access.redhat.com/openshift3/ose-node-problem-detector:<tag>
+$ docker pull registry.access.redhat.com/openshift3/ose-recycler:<tag>
 $ docker pull registry.access.redhat.com/openshift3/ose-template-service-broker:<tag>
```
Comment (ganhuang) [inline install/disconnected_install.adoc]: Should be moved to service catalog section.

File: install/disconnected_install.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: apb-base and apb-tools?

File: install/disconnected_install.adoc (hunk 2 above)
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: jenkins-agent-nodejs-8-rhel and jenkins-agent-nodejs-8-rhel?

File: install/disconnected_install.adoc (hunk 4 above)
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: local-storage-provisioner and logging-eventrouter?

**Comment by ganhuang** [comment]
@kalexand-rh I'm sorry that I'm not quite familiar with all the container images.

Just tell by the image name to give the comments.

File: install/disconnected_install.adoc (hunk 3 above)
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: manila-provisioner and mariadb-apb?

File: install/disconnected_install.adoc (hunk 5 above)
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: mysql-apb?

File: install/disconnected_install.adoc (hunk 7 above)
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: ose-template-service-broker?

**Comment by kalexand-rh** [comment]
@ganhuang, will you check that I'm moving the right images?

File: install/disconnected_install.adoc (hunk 1 above)
Comment (ganhuang) [reply install/disconnected_install.adoc]: right

File: install/disconnected_install.adoc (hunk 2 above)
Comment (ganhuang) [reply install/disconnected_install.adoc]: right

File: install/disconnected_install.adoc (hunk 4 above)
Comment (ganhuang) [reply install/disconnected_install.adoc]: logging-eventrouter only

File: install/disconnected_install.adoc (hunk 3 above)
Comment (ganhuang) [reply install/disconnected_install.adoc]: sorry, mariadb-apb only

File: install/disconnected_install.adoc (hunk 5 above)
Comment (ganhuang) [reply install/disconnected_install.adoc]: right

File: install/disconnected_install.adoc (hunk 7 above)
Comment (ganhuang) [reply install/disconnected_install.adoc]: yes

**Comment by kalexand-rh** [comment]
@ganhuang, IDK if we need to fix this now or with the future work bug, but what do we do with the  service catalog images? Do they go in their own .tar file, like the logging and metrics images, or all in ose3-images.tar?

You know them far better than I do. Thank you!

**Comment by ganhuang** [comment]
@kalexand-rh I revisit the latest document, for the service catalog images, let's make it in ose3-images.tar.

From the user perspective, we should clarify to the users what images they should be synced according their requirements, it's not necessary to sync those images they won't use in their environment, wasting time and disk. 

IMO we definitely need to fix it, but it's too late and too complicated to fix it at this moment.

**Comment by kalexand-rh** [comment]
@ganhuang, ok. I moved all the images that you pull in that section into ose3-images.tar, for now, and we'll make more updates soon. :)

**Comment by ganhuang** [comment]
@kalexand-rh Now LGTM! Thanks!

**Comment by kalexand-rh** [comment]
@ganhuang, thank you so much for the reviews! Merging. :)

**Comment by ganhuang** [comment]
@kalexand-rh Just found that some deprecated images are still present in `Export the OpenShift Container Platform containerized components:`:

```
    registry.access.redhat.com/openshift3/ose \
    registry.access.redhat.com/openshift3/container-engine \
    registry.access.redhat.com/openshift3/node \
    registry.access.redhat.com/openshift3/openvswitch \
```

**Comment by kalexand-rh** [comment]
@ganhuang, I've removed those images.

**Pull request merged**